lib/records/I_RecHttp.h

```diff
@@ -48,18 +49,20 @@ void RecHttpLoadIpMap(const char *name, ///< Name of value in configuration file
 */
 class SessionProtocolSet
 {
-  typedef SessionProtocolSet self; ///< Self reference type.
+  using self_type = SessionProtocolSet; ///< Self reference type.
   /// Storage for the set - a bit vector.
-  uint32_t m_bits;
+  uint32_t m_bits = 0;
 
 public:
   // The right way.
   //  static int const MAX = sizeof(m_bits) * CHAR_BIT;
   // The RHEL5/gcc 4.1.2 way
```

Comment:
Looks like this is going "The right way."

Reply:
Dang, I thought I'd taken those comments out as well.